containers/integration-tests-build/Dockerfile

```diff
@@ -10,16 +10,31 @@ RUN apt-get -y update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     wget \
     software-properties-common
 
-# Intel DCAP libs (this should be removed once we are bundling them in the Conclave runtime along with the
-# last 5 packages installed below)
-RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | tee /etc/apt/sources.list.d/intel-sgx.list
+# Intel SGX libs
 RUN wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
-RUN wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
-RUN add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/ubuntu/20.04/prod focal main"
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intel-sgx.list
+
+RUN apt-get -y update
+
 
 # Disable interactive dialogue
 # Otherwise, a message that requires user input might appear during the installation
+# TODO: Update Conclave to Intel SGX SDK 2.18 and remove package versions: https://r3-cev.atlassian.net/browse/CON-1243
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        az-dcap-client \
-        libsgx-quote-ex \
-        libsgx-dcap-ql
+    sgx-aesm-service=2.17.100.3-focal1 \
```

Comment:
If we have the aesm deamon in the other container, why does it need to be installed here?

Reply:
Uh, this slipped. I forgot to remove these after I realized that I need a (root) container for the AESM. Will remove